server/user-update.js

```diff
@@ -63,6 +63,8 @@ module.exports = function handle({ message = {} }, { ship, hull }) {
     }
 
     if (errors && errors.length > 0) {
+      // TODO: this call can be easily too high volume:
+      // asUser.post(`/${ship.id}/notifications`, { status: "error", message: "Script error" });
```

Comment:
I don't care how we do it. I just care that we are able to show errors on Dashboard and notify customers about them